pygeos/test/test_constructive.py

```diff
@@ -589,3 +589,14 @@ def test_segmentize_none():
 def test_segmentize(geometry, tolerance, expected):
     actual = pygeos.segmentize(geometry, tolerance)
     assert pygeos.equals(actual, geometry).all()
+
+
+@pytest.mark.skipif(pygeos.geos_version < (3, 6, 0), reason="GEOS < 3.6")
+@pytest.mark.parametrize("geometry", all_types)
+def test_minimum_rotated_rectangle(geometry):
```

Comment:
Can you add a test containing the situations that are in the docstring? We don't doctest anymore in all environments.